src/Temporalio/Bridge/include/temporal-sdk-bridge.h

```diff
@@ -0,0 +1,253 @@
+#pragma once
```

Comment:
Note how there is nothing .NET specific here. This whole Rust project should be extracted out of this repo and repurposed for general C FFI use when we need it (e.g. for CGo, JNI, etc).